styles/maps.scss

```diff
@@ -1,82 +1,65 @@
 @use 'variables' as *;
-.title {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  margin: 2rem;
-}
 
-.title div{
-  display: flex;
-  justify-content: center;
-  align-items: baseline;
+.title div i {
+  padding-left: 1rem;
 }
-
-.add{
+.add {
   color: $color;
   background-color: transparent;
   font-weight: bolder;
   border: none;
   font-size: large;
-}
-body, html {
-  height: 80%;
-  width: 100%;
-}
-// main {
-//   display: flex;
-//   flex-direction: row;
-//   justify-content: space-between;
-//   align-items: start;
-//   height: 100%;
+  cursor: pointer;
 
-// }
+  &:hover {
+    opacity: 0.8;
+  }
+}
 .pins {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 2;
+  width: 55%;
+  margin: 0 30px 0 0;
 }
 .info-window {
   color: $primary;
 }
 #map {
-  height: 100%;
-  min-height: 400px;
-  flex-grow: 5;
-  min-width: 300px;
-  margin-bottom: 1.5rem;
-}
-.pin{
-  margin: 1rem;
-  margin-left: 0;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  min-width: 0;
-  border-bottom: 1px solid $color;
-  padding-bottom: 1rem;
-  flex-grow: 2;
+  height: 80vh;
+  width: 45%;
 }
-.description {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 2;
-  margin-right: 1rem;
-  min-width: none;
-}
-.pinImg {
-  border-radius: 1rem;
-  border: 1px solid $color;
-  flex-grow: 1;
-  max-width: 120px;
-  min-width: 80px;
-  max-height: 120px;
-  min-height: 80px;
+.pin {
+  border-bottom: 1px solid $light;
+  padding: 15px;
+
+  &:hover {
```

Comment:
You can nest code when using SCSS. All the styles related to <div class='pin'> goes under here